_pages/fsa/dynamicforms/styling.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Dynamicforms Styling
+---
+
+Dynamic forms Library uses Bootstrap 4 classes for styling. **Custom** CSS classes can be added in four places while defining JSON form definition through Back-Office.
+
+Custom CSS class can be defined for:
```

Comment:
What do you think about referencing the interfaces where custom css can be defined?
For example provide a link to the documented interface when that PR gets merged?

Reply:
Reference is added to the form json structure explanations